src/components/projects/CreateProjectForm.vue

```diff
@@ -820,9 +864,14 @@ export default {
       max_fits_header_length: 68,
       generic_filter_list: ['Lum', 'Red', 'Green', 'Blue', 'HA', 'O3', 'S2', 'EXO'],
       generic_camera_areas: [
-        '600%', '500%', '400%', '300%', '220%', '133%',
-        'FULL', 'SQUARE', '71%', '50%', '35%', '25%', '12%'
+        'Mosaic deg.', 'Mosaic arcmin.', 'Full', 'Big sq.',
+        'Small sq.', '71%', '50%', '35%', '25%', '18%', '12.5%', '9%', '6%'
       ],
+      minWidth: -4.5,
```

Comment:
It may be overkill here but it is often nice to include the units in the variable name, e.g. minWidthDegrees

Reply:
would this be okay even if arcmin is selected?

Reply:
It makes it even clearer because it adds some context to the '*60' because it would now be 'minWidthDegrees * 60' which is clearly something in arcminutes.
Even better would be to define a constant 'degreesToArcminutes = 60'. Then it would be 'minWidthDegrees * degreesToArcminues' which is totally unambiguous and devoid of magic numbers.

Reply:
gotcha! Good point, making the changes now

Reply:
Done